bedrock/firefox/templates/firefox/os/devices.html

```diff
@@ -1,2882 +0,0 @@
-                    </td>
-                  </tr>
-                  <tr>
-                    <th>{{ _('Memory') }}</th>
-                    <td>
-                      <dl>
-                        <dt>{{ _('ROM') }} *</dt>
-                        <dd>{{ _('256 MB') }}</dd>
-
-                        <dt>{{ _('RAM') }}</dt>
-                        <dd>{{ _('128 MB') }}</dd>
-
-                        <dt>{{ _('Expansion slot') }}</dt>
-                        <dd>{{ _('MicroSD up to 4 GB') }}</dd>
-                      </dl>
-
-                      <small>* {{ memory_disclaimer }}</small>
-                    </td>
-                  </tr>
-                  <tr>
-                    <th>{{ _('Battery') }}</th>
-                    <td>
-                      <dl>
-                        <dt>{{ _('Capacity') }}</dt>
-                        <dd>{{ _('1400 mAh') }}</dd>
-
-                        <dt>{{ _('Talk time') }}</dt>
-                        <dd>{{ _('Up to 6 h') }}</dd>
-                      </dl>
-                    </td>
-                  </tr>
-                  <tr>
-                    <th>{{ _('Networks') }}</th>
-                    <td>
-                      {{ _('GPRS/EDGE') }}
-                      <br>
-                      {{ _('GSM 900/1800 MHz') }}
-                    </td>
-                  </tr>
-                  <tr>
-                    <th>{{ _('Wireless') }}</th>
-                    <td>
-                      {{ _('Bluetooth') }}
-                      <br>
-                      {{ _('Wi-Fi') }}
-                    </td>
-                  </tr>
-                  <tr>
-                    <th>{{ _('Ports') }}</th>
-                    <td>
-                      {{ _('3.5 mm stereo headset jack') }}
-                    </td>
-                  </tr>
-                </table>
-              </div>{#--/.specifications--#}
-            </div>{#--/.pager-content--#}
-          </div>{#--/.container--#}
-        </section>{#--/#spice_fireonemifx1--#}
-
-        <section class="device-detail smartphone-detail" id="symphony_gofoxf15">
-          <div id="symphony_gofoxf15-pager" class="container pager pager-with-tabs pager-no-history">
-            <header class="device-detail-header">
-              <h3 class="device-title">{{ _('Symphony GoFox F15') }}</h3>
-
-              <ul class="pager-tabs">
-                <li><a href="#p1" data-label="Features">{{_('Features')}}</a></li>
-                <li><a href="#p2" data-label="Specifications">{{_('Specifications')}}</a></li>
-              </ul>
-
-              <button class="device-detail-close">{{ _('Close') }}</button>
-            </header>
-
-            <div class="pager-content">
-              <div class="device-details features pager-page default-page" id="symphony_gofoxf15-features">
-                <div class="feature-group logo-colors">
-                </div>
-
-                <div class="feature-group image">
-            
```

Comment:
I can't figure out which one you're referring to here. We may need to discuss this bit more.